src/tests/stringy.spec.ts

```diff
@@ -53,4 +59,63 @@ describe('Stringy methods', () => {
       expect(result).to.equal(true);
     });
   });
+
+  describe('isSuffix', () => {
+    const cashew = new Stringy('cashew');
+    const cashewWithStartingSpace = new Stringy(' cashew');
+    const casHEW = new Stringy('HEW');
+    const ring = new Stringy('ring')
+
+    it('This should return false for s in as in cashew', () => {
+      const result = cashew.isSuffix('s');
+      expect(result).to.equal(false);
+    });
+
+    it('This should return false for sew as in cashew', () => {
+      const result = cashew.isSuffix('sew');
+      expect(result).to.equal(false);
+    });
+
+    it('This should return false for HEW as in cashew with isCaseSensitive as true', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'This·should·return·false·for·HEW·as·in·cashew·with·isCaseSensitive·as·true'` with `"This·should·return·false·for·HEW·as·in·cashew·with·isCaseSensitive·as·true"`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
    it("This should return false for HEW as in cashew with isCaseSensitive as true", () => {
```